plugins/com.google.cloud.tools.eclipse.appengine.newproject/src/com/google/cloud/tools/eclipse/appengine/newproject/standard/AppEngineStandardProjectWizard.java

```diff
@@ -91,17 +63,4 @@ public boolean performFinish() {
     }
     return accepted;
   }
-
-
-  private class DependencyValidator implements IRunnableWithProgress {
-
-    private IStatus result;
-
-    @Override
-    public void run(IProgressMonitor monitor)
-        throws InvocationTargetException, InterruptedException {
-      result = resolverService.checkRuntimeAvailability(AppEngineRuntime.STANDARD_JAVA_7, monitor);
```

Comment:
I was going to remove it as part of #2999, since it wasn't being performed under a scheduling rule too, but left it in as it also tested that the rules were now being obtained.